packages/frontend-shared/src/gql-components/HeaderBarContent.cy.tsx

```diff
@@ -378,6 +378,10 @@ describe('<HeaderBarContent />', { viewportWidth: 1000, viewportHeight: 750 }, (
             },
             render: (gqlVal) => <div class="border-current border-1 h-700px resize overflow-auto"><HeaderBarContent gql={gqlVal} show-browsers={true} allowAutomaticPromptOpen={true} /></div>,
           })
+
+          cy.tick(2000)
```

Comment:
Could we add a comment with the intention behind this tick? Would tests fail if we removed it?